src/lib/Context.ts

```diff
@@ -88,7 +100,11 @@ export const readConfig = (path: string): ConfigJson => {
   return filteredConfig
 }
 
-export const createSignerFromPath = async (path: string): Promise<Signer> => {
+export const createSignerFromPath = async (path: string | undefined): Promise<Signer> => {
+  if (!path) {
+    throw new Error('No keypair path provided')
```

Comment:
throw new Error('No keypair specified in config or args. Please set a keypair path in your config file or use --keypair flag.')